datafile-handler/src/androidTest/java/com/optimizely/ab/android/datafile_handler/DatafileLoaderTest.java

```diff
@@ -170,4 +170,29 @@ public void warningsAreLogged() throws IOException {
         verify(logger).warn("Unable to save new datafile");
         verify(datafileLoadedListener, atMost(1)).onDatafileLoaded("{}");
     }
+
+    @Test
+    public void debugLogged() throws IOException {
+        final ListeningExecutorService executor = MoreExecutors.newDirectExecutorService();
+        Cache cache = mock(Cache.class);
+        datafileCache = new DatafileCache("debugLogged", cache, logger);
+        DatafileLoader datafileLoader =
+                new DatafileLoader(datafileService, datafileClient, datafileCache, executor, logger);
+
+        when(client.execute(any(Client.Request.class), anyInt(), anyInt())).thenReturn("{}");
+        when(cache.exists(datafileCache.getFileName())).thenReturn(true);
+        when(cache.delete(datafileCache.getFileName())).thenReturn(false);
+        when(cache.save(datafileCache.getFileName(), "{}")).thenReturn(false);
+
+        datafileLoader.getDatafile("debugLogged", datafileLoadedListener);
+        datafileLoader.getDatafile("debugLogged", datafileLoadedListener);
+        try {
+            executor.awaitTermination(5, TimeUnit.SECONDS);
+        } catch (InterruptedException e) {
+            fail();
+        }
+
+        verify(logger).debug("Last download happened under 1 minute ago. Throttled to be at least 1 minute apart.");
+        verify(datafileLoadedListener, atMost(2)).onDatafileLoaded("{}");
```

Comment:
Let's also check if it's called at least once
```suggestion
        verify(datafileLoadedListener, atMost(2)).onDatafileLoaded("{}");
        verify(datafileLoadedListener, atLeast(1)).onDatafileLoaded("{}");
```